### ISSUE TITLE
diaNN converter

### ISSUE THREAD

**Issue by mstaniak** [Vitek-Lab/MSstatsConvert#65]

**Comment by mstaniak** [comment]
hi, diann-converter branches on MSstatsConvert and MSstats repositories have a working version of this converter

**Comment by chscho** [comment]
Hi Mateusz

Thank you for releasing this DIANN-Converter. We're currently testing  DIANN and also its post-processing with MSstats. I've just noticed, that with the most recent version of MSstatsConvert (a80befb), following error pops up:

```
INFO  [2022-11-04 11:51:26] ** Raw data from DIANN imported successfully.
Error in getClass(Class, where = topenv(parent.frame())) : 
  “MSstatsDIANNFiles” is not a defined class
```
...if I roll back to MSstatsConvert (b1481f4) everything is working fine again. 

(R4.2.1; MSstatsConvert_1.5.1 (a80befb / b1481f4); MSstats_4.4.1 (55f3973); x86_64-pc-linux-gnu (64-bit))

Further, I've noticed that you've indicated using "ProteinsIds" as protein_id_column. I think this is quite dangerous due to the (less common) protein inference approach of DIA-NN (https://github.com/vdemichev/DiaNN/issues/22#issuecomment-609491034) resulting in multiple Protein-Groups. Hence I would rather recommend taking the "ProteinGroup" column instead (with activated heuristic protein interference)...

Best,
Christian
